cmd/juju/user/login.go

```diff
@@ -109,17 +111,17 @@ func (c *loginCommand) SetFlags(fset *gnuflag.FlagSet) {
 	c.ControllerCommandBase.SetFlags(fset)
 	fset.StringVar(&c.controllerName, "c", "", "Controller to operate in")
 	fset.StringVar(&c.controllerName, "controller", "", "")
-	fset.BoolVar(&c.forceUser, "u", false, "log into the current controller as a local user")
-	fset.BoolVar(&c.forceUser, "user", false, "")
+	fset.StringVar(&c.username, "u", "", "log in as this local user")
+	fset.StringVar(&c.username, "user", "", "")
```

Comment:
As they both refer to the same variable, they're treated as aliases and the same description is used for both (taken from the shortest flag as per the gnuflag docs).